src/App.tsx

```diff
@@ -17,18 +47,26 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onChangeQuery={setQuery}
+                onChangeType={setTodoType}
+                query={query}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!todos
```

Comment:
Better to create `isLoading` state and change it to `true` before API call and to `false` after
Because now if you get empty response you will loader all the time